suites/guppy/nestedQueryTest.js

```diff
@@ -0,0 +1,242 @@
+/*
+ * Prerequisites:
+ * 1. Data in the commons are ETLed
+ * 2. Data has a nested structure. Take qa-mickey as example:
+ * "patient" doc has nested prop "ActionableMutations"
+ * 3. Guppy is up, take qa-mickey as example:
+          guppy version entry: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/manifest.json#L23
+          config: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/manifest.json#L96-L104
+          related portal config: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/portal/gitops.json#L223-L228
+ * 4. User got access to the website and the data, and could visit `/query` page
+ */
+Feature('Support query for nested data - https://github.com/uc-cdis/guppy/pull/52');
```

Comment:
We usually reference the JIRA url here, right?
e.g.,
https://github.com/uc-cdis/gen3-qa/pull/175/files#diff-00a19f5fc7c95ed6f4a05ff5721adc5cR1